<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Bullet</class>
 <widget class="QDialog" name="Add_Bullet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Bullet</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Kind</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Bullet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Velocity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>BC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Drag Function</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Kind">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Weight">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Bullet">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Velocity">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_BC">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_DragFunction">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>G1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RA4</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>gr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>libre</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>m/sec</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>lb/ùëñùëõ^2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBox_OK">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBox_Cancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
